Makefile

```diff
@@ -246,6 +246,7 @@ $(DOC_LOCATION):
 	@echo "Building docs..."
 	@bash -c '$(CONDA_CMD) \
 		sphinx-apidoc -o "$(APP_ROOT)/docs/" "$(APP_ROOT)/$(APP_NAME)"; \
+		sphinx-apidoc -o "$(APP_ROOT)/docs/" "$(APP_ROOT)/tests"; \
```

Comment:
The reason I added docs for the tests is because we mentioned adding a link between the Geoserver tests and the documentation on the Geoserver handler component, where it explains the different design choices.

I ended up just putting one single link, so I can always remove this, and change the description in [test_geoserver.py::TestGeoserverPermissions](https://github.com/Ouranosinc/cowbird/blob/4d6c2d160a20d2904a74fb1c15d46aab547a7f91/tests/test_geoserver.py#L249) to mention the doc without necessarily using a link?

Reply:
Oh, ok. Fine to leave it in then. Just make sure that the `sphinx-autoapi` properly generates the source code since `"$(APP_ROOT)/$(APP_NAME)` was removed.